core/mapi/cache/namedPropCache.h

```diff
@@ -7,8 +7,11 @@ namespace cache
 	{
 		// Returns a vector of NamedPropCacheEntry for the input tags
 		// Sourced directly from MAPI
-		_Check_return_ std::vector<std::shared_ptr<namedPropCacheEntry>>
-		GetNamesFromIDs(_In_ LPMAPIPROP lpMAPIProp, _In_opt_ LPSPropTagArray* lppPropTags, ULONG ulFlags);
+		_Check_return_ HRESULT GetNamesFromIDs(
+			_In_ LPMAPIPROP lpMAPIProp,
+			_In_opt_ LPSPropTagArray lpPropTags,
```

Comment:
I think I changed this based on some intellisense recommendations to const LPSPropTagArray . If so, need to update the declaration also.